core/src/main/java/tc/oc/pgm/goals/TouchableGoal.java

```diff
@@ -192,15 +195,13 @@ protected void sendTouchMessage(@Nullable ParticipantState toucher, boolean incl
     Component message = getTouchMessage(toucher, false);
     Audience.console().sendMessage(message);
 
-    if (!showEnemyTouches()) {
-      message = text().append(toucher.getParty().getChatPrefix()).append(message).build();
-    }
-
-    for (MatchPlayer viewer : getMatch().getPlayers()) {
-      if (shouldShowTouched(toucher.getParty(), viewer.getParty())
```

Comment:
Pretty sure this is a behavior change with matches with +2 teams. Before it would show this to all players in a team that had touched, while now it will only show to the team that touched.
Eg, a 4-team DTC map:
Blue touches red core <- Blue is notified
Green touches red core <- Before, both green and blue are notified, now only green is.